weatherClock.py

```diff
@@ -98,42 +93,10 @@ def get_mouse_click_coor(x, y):
 
     hourTouched = -1
 
-    if (touchInBox(cursor_x, cursor_y, hour1_x, hour1_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 1 WAS TOUCHED !")
-        hourTouched = 1
-    elif (touchInBox(cursor_x, cursor_y, hour2_x, hour2_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 2 WAS TOUCHED !")
-        hourTouched = 2
-    elif (touchInBox(cursor_x, cursor_y, hour3_x, hour3_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 3 WAS TOUCHED !")
-        hourTouched = 3
-    elif (touchInBox(cursor_x, cursor_y, hour4_x, hour4_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 4 WAS TOUCHED !")
-        hourTouched = 4
-    elif (touchInBox(cursor_x, cursor_y, hour5_x, hour5_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 5 WAS TOUCHED !")
-        hourTouched = 5
-    elif (touchInBox(cursor_x, cursor_y, hour6_x, hour6_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 6 WAS TOUCHED !")
-        hourTouched = 6
-    elif (touchInBox(cursor_x, cursor_y, hour7_x, hour7_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 7 WAS TOUCHED !")
-        hourTouched = 7
-    elif (touchInBox(cursor_x, cursor_y, hour8_x, hour8_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 8 WAS TOUCHED !")
-        hourTouched = 8
-    elif (touchInBox(cursor_x, cursor_y, hour9_x, hour9_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 9 WAS TOUCHED !")
-        hourTouched = 9
-    elif (touchInBox(cursor_x, cursor_y, hour10_x, hour10_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 10 WAS TOUCHED !")
-        hourTouched = 10
-    elif (touchInBox(cursor_x, cursor_y, hour11_x, hour11_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 11 WAS TOUCHED !")
-        hourTouched = 11
-    elif (touchInBox(cursor_x, cursor_y, hour12_x, hour12_y, hourlyTouchSize, hourlyTouchSize)):
-        print("hour 12 WAS TOUCHED !")
-        hourTouched = 12
+    for i in range(0, 12):
```

Comment:
Would it make sense to iterate over `hours`?

I also think there should be `break` statement to avoid always performing 12 comparisons.

How about something like this?

```python
for hourTouched, hour in enumerate(hours, start=1):
  if (touchInBox(cursor_x, cursor_y, hour[0], hour[1], hourlyTouchSize, hourlyTouchSize)):
    print(f"{hourTouched} WAS TOUCHED !")
    # No need to update hourTouched
    break
```